install-dev.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+# Check if pip is installed, and install it if necessary
+if ! command -v pip >/dev/null 2>&1; then
+    echo "---- Installing pip ----"
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
Is that possible to automatically remove the pip install file ?

Reply:
Yeah. I could add rm get-pip.py after that.